### ISSUE TITLE
Make testable live website

### ISSUE THREAD

**Issue by Eevinjennings** [hackforla/access-the-data#187]
### Overview
In order to progress with research and design choices, we need a basic website for users to interact with. I will make this using some tools. 

### Action Items
- [ ] Decide which platform I will use. 
- [ ] See if I'm able to share design editing options with teammates. 
- [ ] Make the site.
- [ ] Work with @Aparna1Gopal and @lrchang2 to integrate Microsoft Clarity, Google Analytics, and/or HotJar embed codes.

### Resources/Instructions
- None yet

**Comment by Aparna1Gopal** [comment]
@Eevinjennings Is this something we can look at? To make the process of creating a live website simpler? Embedding our existing Figma prototype into a live website tool? 

https://www.figma.com/prototyping/
Under: Other ways to use Figma prototypes
Interact with live prototypes in Notion or Dropbox
Integrate with third-party prototyping tools

https://www.figma.com/blog/introducing-figmas-live-embed-kit/

**Comment by Eevinjennings** [comment]
> @Eevinjennings Is this something we can look at? To make the process of creating a live website simpler? Embedding our existing Figma prototype into a live website tool?
> 
> https://www.figma.com/prototyping/ Under: Other ways to use Figma prototypes Interact with live prototypes in Notion or Dropbox Integrate with third-party prototyping tools
> 
> https://www.figma.com/blog/introducing-figmas-live-embed-kit/

@Aparna1Gopal Good idea, but this only allows you to embed the viewable file like a PDF. While we could technically track users' behaviors with heatmapping, I'm not sure what we'd be able to get out of the analytics. That being said, we can totally try it! I've put an example of the embed on the site if you want to check it out: https://access-the-data.webflow.io/figma-embed

**Comment by Aparna1Gopal** [comment]
@Eevin Good job! I am just excited seeing it live! At least it is interactive and gives us the framework to test with outside users, unmoderated.  We could start with testing on small tasks and features for the user flows and IA. Thank You! 

Now we just need to figure out how we could embed Hotjar/Analytics/other testing tools to this. 

Once the new direction for data sources (CKAN) are decided and this design copy keeps getting updated, we could just update our embed and use it. Like an iterative process. What do you think? 

Also, is the embed process from Figma to Webflow difficult? If you could please list out the process in this GitHub issue, like a guide, any of us could do it moving forward as and when the design is updated. Your thoughts?

**Comment by Eevinjennings** [comment]
@Aparna1Gopal It's not difficult at all! Webflow has some great tutorials on how to do a custom embed with Figma 👍

As for Analytics and Clarity, we will need to make a code that I can paste into the HMTL on each page. I learned how to do it by following the step-by-step instructions on Analytics and Clarity 😀 I'll just need the embed code for the HTML head once you've got it setup.

**Comment by Aparna1Gopal** [comment]
@Eevinjennings Thanks! I will check out these Webflow tutorials for the custom embed :)

I have no clue how 'the embed code for the HTML head' would work including 'making the code.' I will need to do a deep dive into the step-by-step tutorial as well!

**Comment by mxajPrice** [comment]
Need to ask @Eevinjennings  about access to this.
